src/test/java/com/qualimente/training/addressbook/ControllerTest.java

```diff
@@ -14,14 +14,21 @@
   @Test
   public void constructor_should_store_arguments(){
     AddressDAO AddressDAO = makeAddressDAO();
-    Controller controller = new Controller(AddressDAO);
+    Controller controller = new Controller(AddressDAO, new LocationDataValidator());
```

Comment:
The addressDao is constructed using a separate private method.  Consider using the same patter for the LocationDataValidator for consistency.